tests/unit/core/metrics/test_mlc_map.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Unit tests for PCK metric."""
```

Comment:
Could you rename? And also the class name

Reply:
I think it was for key point detection

Reply:
Done